src/components/ScheduleGenerate/RequirementCourses.vue

```diff
@@ -26,19 +26,19 @@
       </button>
     </div>
     <div v-if="showDropdown">
-      <p class="requirement-header-subtext">Please select a major requirement.</p>
+      <p class="requirement-header-subtext">Please select a major/minor requirement.</p>
```

Comment:
Also added this as the possible requirement groups include those that apply to your minor.